src/Microsoft.Health.Dicom.Core/Messages/CustomTag/AddCustomTagResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.CustomTag
+{
+    public class AddCustomTagResponse
+    {
+        private readonly IEnumerable<CustomTagEntry> _customTags;
+        private readonly string _job;
+
+        public AddCustomTagResponse(IEnumerable<CustomTagEntry> customTags, string job)
+        {
+            _customTags = customTags;
+            _job = job;
+        }
+
+        public IEnumerable<CustomTagEntry> CustomTags { get => _customTags; }
```

Comment:
nit: you can just convert these and the others like it to auto-properties

e.g. remove the private variables and replace this line with something like
```csharp
public IEnumerable<CustomTagEntry> CustomTags { get; }
```

Then in the constructor you can change `_customTags = customTags;` to `CustomTags = customTags`

Reply:
resolved.